src/core/include/openvino/op/col2im.hpp

```diff
@@ -42,6 +42,19 @@ class OPENVINO_API Col2Im : public ov::op::Op {
     const Shape& get_pads_begin() const;
     const Shape& get_pads_end() const;
 
+    void set_strides(const Strides& strides) {
+        m_strides = strides;
+    };
+    void set_dilations(const Strides& dilations) {
+        m_dilations = dilations;
+    };
+    void set_pads_begin(const Shape& pads_begin) {
+        m_pads_begin = pads_begin;
+    };
+    void set_pads_end(const Shape& pads_end) {
+        m_pads_end = pads_end;
+    };
+
```

Comment:
do we need this set methods? I think we can just use constructor..

Reply:
Generally set attributes are being used for others. I think we donot need to remove it.